jupyterhub/app.py

```diff
@@ -1311,6 +1311,13 @@ def cleanup(self):
         """Shutdown our various subprocesses and cleanup runtime files."""
```

Comment:
Suggestion for Line 1330: Replace SUS with single user servers

 # clean up proxy while SUS are shutting down